input/cql/IntravesicalBacillusCalmetteGuerinForBladderCancerFHIR.cql

```diff
@@ -80,6 +84,9 @@ define "July 1 of Year Prior to the Measurement Period":
 define "June 30 of the Measurement Period":
   DateTime((year from start of "Measurement Period"), 6, 30, 23, 59, 59, 0, 0)
 
+/*
+@QICore6Uplift: Update Condition to ConditionEncounterDiagnosis
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis